backend/api/Database/Models/Robot.cs

```diff
@@ -20,7 +20,7 @@ public Robot()
             Pose = new Pose();
         }
 
-        public Robot(CreateRobotQuery createQuery, Installation installation, Area? area = null)
+        public Robot(CreateRobotQuery createQuery, Installation installation, RobotModel model, Area? area = null)
```

Comment:
Could consider putting this as part of CreateRobotQuery

Reply:
That's a bit difficult since it needs to be passed in the Swagger endpoint in that case